libr/debug/p/native/w32.c

```diff
@@ -729,6 +730,9 @@ static int w32_dbg_wait(RDebug *dbg, int pid) {
 			break;
 		case EXCEPTION_DEBUG_EVENT:
 			switch (de.u.Exception.ExceptionRecord.ExceptionCode) {
+#if __MINGW64__
+			case 0x4000001f:
```

Comment:
is a special exception for win32 subsystem, dont know if mingw have it defined in their sdk.